<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminInfo</class>
 <widget class="QWidget" name="AdminInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="fm_personal_infor">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>471</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lb_major_txt">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_number">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>学号</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_username">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_credit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>信誉分</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_sex_txt">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_number_txt">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_major">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>专业</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_sex">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>性别</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_credit_txt">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_username_txt">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="line_confirmPassword">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>600</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_modifyPassword">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>350</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_password">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_password">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>480</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>50</y>
     <width>10</width>
     <height>770</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_modifyPassword">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>710</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_confirmPassword">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>540</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_major">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>520</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_number_2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>160</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>学号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_number">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>220</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_mojor_2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>580</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>邮箱</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_name">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>340</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_modifyInfo">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>90</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>修改信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_modifyInfo">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>710</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>修改信息</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_major_2">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>640</y>
     <width>400</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_name">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>280</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box_sex">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>400</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_mojor">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>460</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>专业</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_sex_2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>400</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
